common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -142,9 +137,7 @@ define('video/04_video_full_screen.js', [], function () {
         this.videoFullScreen.fullScreenEl
             .find('.icon')
                 .removeClass('fa-compress')
-                .addClass('fa-arrows-alt')
-                .find('.control-text')
-                    .text(gettext('Fill browser'));
+                .addClass('fa-arrows-alt');
```

Comment:
If you change the title below to "Exit full browser", it will need to be changed back here.